notes/index.html

```diff
@@ -95,21 +95,21 @@ <h3>"Copykitten"</h3>
 
-<p>Je suis seulement dans ma vingtaine, je ne sais pas pourquoi je donne toujours l'impression d'&ecirc;tre un vieil aigri dirigeant mon poing vers un nuages. Mais, oui, lisez le livre de 1985 de Neil Postman, <a href="https://www.amazon.com/Amusing-Ourselves-Death-Discourse-Business/dp/014303653X">Amusing Ourselves To Death</a>. C'est sur la mani&egrave;re de laquelle les technologies de la communication transforment subtilement notre culture ("Le moyen est le message") et comment la technologie orient&eacute;e vers des informations rapides (TV en 1985, m&eacute;dia sociaux maintenant) nous transforme tous and des personnes incons&eacute;quentes et avec une pens&eacute;e &agrave; court terme.</p>
+<p>J'ai seulement la vingtaine, je ne sais pas pourquoi je donne toujours l'impression d'&ecirc;tre un vieil aigri levant mon poing vers un nuage. Mais, oui, lisez le livre de 1985 de Neil Postman, <a href="https://www.amazon.com/Amusing-Ourselves-Death-Discourse-Business/dp/014303653X">Amusing Ourselves To Death</a>. C'est sur la mani&egrave;re dont les technologies de la communication transforment subtilement notre culture ("Le moyen est le message") et comment la technologie orient&eacute;e vers des informations rapides (TV en 1985, m&eacute;dia sociaux maintenant) nous transforme tous en des personnes incons&eacute;quentes et avec une pens&eacute;e &agrave; court terme.</p>
 
 <p>Bien que je ne sois pas d'accord avec <em>tout</em> ce qui est dans son livre, il est &eacute;tonamment clairvoyant pour 1985. Sa diatribe contre Sesame Street est plut&ocirc;t &eacute;trange, par contre.</p>
 
-<h3>"Build relationships. Find win-wins. Communicate clearly." | "Construisez des relations, Trouvez des gagnant-gagnants. Communiquez clairement"</h3>
+<h3>"Build relationships. Find win-wins. Communicate clearly." | "Construisez des relations. Trouvez des gagnant-gagnants. Communiquez clairement."</h3>
 
 <p>Donner des conseils peut sembler condescendant, alors laissez-moi &ecirc;tre tout &agrave; fait honn&ecirc;te : Je <em>n'ai pas</em> la moindre id&eacute;e de ce qu'il convient de faire, personnellement. J'ai du mal &agrave; construire des amiti&eacute;s, tout particuli&egrave;rement avec des gens d'un avis politique diff&eacute;rent du mien. J'oublie parfois de d&eacute;velopper des relations gagnant-gagnant et tombe dans un mode de pens&eacute;e "eux contre nous". Et &agrave; en juger par mes propos, je suis encore tr&egrave;s mauvais pour ce qui est de communiquer clairement.</p>
 
-<p>Mais je veux m'am&eacute;liorer. Ces choses sont <em>dures</em>. Construire la paix &emp; avoir confiance dans le monde, partout, est <em>dur</em>.</p>
+<p>Mais je veux m'am&eacute;liorer. Ces choses sont <em>dures</em>. Construire la paix &amp; avoir confiance dans le monde, partout, est <em>dur</em>.</p>
 
-<p>Et c'est pourquoi ça en vaut le coup. &lt;3</p>
+<p>Et c'est pourquoi ça vaut le coup. &lt;3</p>
```

Comment:
Not sure this should be changed, if someone else has an opinion on the matter, I'd gladly take it.

Reply:
Felt like a register mismatch: I'd say either "ça vaut le coup" or "ça en vaut la peine". But it's a minor thing. Do as you like; you own the URL in the end.

Reply:
"ça en vaut la peine" sounds good to me.
If nobody comments, I'll accept your PR as soon as we are done with the other comment.